demo/index.html

```diff
@@ -68,12 +68,11 @@
           border-radius: 5px;
           border-width: 1px;
         }
-        
         button.selected {
           background-color: #b3e5fc;
         }
       </style>
-      <template is="dom-bind">
+      <dom-bind><template is="dom-bind">
         <template is="dom-repeat" items="[[containers]]">
```

Comment:
Does this `dom-repeat` require a wrapper? I know that you don't need them within a `dom-module`'s template but I can't remember if they're necessary in another `dom-*`'s template like this one.